packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -469,6 +469,7 @@ export const versionedComponents = {
     General: {
       content: {
         '11.1.0': 'data-testid Panel editor content',
+        '8.0.0': 'Panel editor content',
         [MIN_GRAFANA_VERSION]: 'Panel editor content',
```

Comment:
Can we remove the key value pair for `MIN_GRAFANA_VERSION`?

Reply:
I left it there just in case, but you are right we can kick it out. Done via https://github.com/grafana/grafana/pull/97210/commits/b02337076da4bb1b84c82d2f5cb533fb9cce8187